run.sh

```diff
@@ -34,11 +34,15 @@ prepare () {
 
   # These directiories essentially define the structure of our API endpoints.
   # TODO: These should perhaps live in python, near the schemas (defined in api/)?
-  STATES_DIR="${API_OUTPUT_DIR}/";
+
+  INPUT_BASE_DIR="${API_OUTPUT_DIR}/";
+
+  API_OUTPUT_COUNTIES = "${API_OUTPUT_DIR}/counties"
+  API_OUTPUT_STATES = "${API_OUTPUT_DIR}/states"
```

Comment:
So to match the v1 API spec, this should still have a `us/` in the path, no? https://www.dropbox.com/scl/fi/o4bec2kz8dkcxdtabtqda/CAN-V1-API-Draft.paper